treadmill-aws/Makefile

```diff
@@ -54,7 +54,8 @@ sbin: \
 	$(VENV)/sbin/opts.sh \
 	$(VENV)/sbin/start.sh \
 	$(VENV)/sbin/start_node.sh \
-	$(VENV)/sbin/start_master.sh
+	$(VENV)/sbin/start_master.sh \
```

Comment:
why do you need start_master? it is obsolete, we do not have masters in aws, all master procs run inside containers.

Reply:
Oversight, removed.